src/fragments/lib/datastore/ios/data-access/delete-snippet.mdx

```diff
@@ -1,3 +1,5 @@
+To delete an item, simply pass in an instance.
```

Comment:
suggestion: *simply pass in an instance of model object*

Reply:
i think object == instance, how about "To delete an item, simply pass in an instance of a model" or "To delete an item, simply pass in a model instance"?

Reply:
yes, this sounds good!